sorc/orog_mask_tools.fd/orog_netcdf.fd/ramp.fd/ramp.f90

```diff
@@ -0,0 +1,164 @@
+ program ramp_netcdf
+
+! Convert the antarctica RAMP data to netcdf.
+
+ use netcdf
+
+ implicit none
+
+ integer*4, parameter :: idim=43201
+ integer*4, parameter :: jdim=3601
+ integer*4, parameter :: idim_p1=43202
+ integer*4, parameter :: jdim_p1=3602
+
+ character(len=150) :: filenetcdf, fileraw
+
+ integer :: i, istat, ncid, status, dim_i, dim_j
+ integer :: dim_ip1, dim_jp1
+ integer :: id_lon, id_lat, id_data
+ integer :: id_lat_corner, id_lon_corner
+
+ real(kind=4), allocatable :: topo(:,:)
+
+ real(kind=8), allocatable :: lats(:), lons(:)
+ real(kind=8), allocatable :: lats_corner(:), lons_corner(:)
+ real(kind=8)    :: lat11, lon11, dx, dy
+
+ dx = 1.0_8/120.0_8
+ dy = 1.0_8/120.0_8
+
+ lat11 =  -(90.0_8) + dy*0.5_8
+ lon11 = 0.0_8 + dx*0.5_8
+
+ allocate(lons(idim),lats(jdim),topo(idim,jdim))
+ allocate(lons_corner(idim_p1),lats_corner(jdim_p1))
+
+ do i = 1, idim
+   lons(i) = real((i-1),8) * dx + lon11
+   if (lons(i) > 360.0_8) lons(i) = 360.0_8 - lons(i)
+   print*,'lon ',i,lons(i)
+ enddo
+
+ do i = 1, jdim
+   lats(i) = real((i-1),8) * dy + lat11
+   print*,'lat ',i,lats(i)
+ enddo
+
+ lat11 =  -90.0_8
+ lon11 = 0.0_8
+
+ do i = 1, idim_p1
+   lons_corner(i) = real((i-1),8) * dx + lon11
+   if (lons_corner(i) > 360.0_8) lons_corner(i) = 360.0_8 - lons_corner(i)
+   print*,'lon_corner ',i,lons_corner(i)
+ enddo
+
+ do i = 1, jdim_p1
+   lats_corner(i) = real((i-1),8) * dy + lat11
+   print*,'lat_corner ',i,lats_corner(i)
+ enddo
+
+ fileraw="/scratch1/NCEPDEV/global/glopara/fix/raw/orog/thirty.second.antarctic.new.bin"
+
+ open(11, file=trim(fileraw), form='unformatted', access='sequential', iostat=istat)
+ print*,'iostat on open ',istat
+ read(11, iostat=istat) topo
+ print*,'iostat on read ',istat
+ if (istat /= 0) stop 99
+ close(11)
+
+ print*,'topo ', maxval(topo),minval(topo)
+
+ print*,'point 1/1 ',topo(1,1)
+ print*,'point idim/jdim ',topo(idim,jdim)
+
+ filenetcdf="./thirty.second.antarctic.new.nc"
```

Comment:
as above I would suggest a more descriptive and consistent file name like:
topography.antarctica.ramp.30s.nc

Reply:
Done.